lib/SPIRV/libSPIRV/SPIRVValue.h

```diff
@@ -47,6 +47,8 @@
 #include "SPIRVEntry.h"
 #include "SPIRVType.h"
 
+#include "llvm/ADT/APInt.h"
```

Comment:
It is good that we don't have this include in `SPIRVModule.h` anymore, but `SPIRVValue.h` is a pretty "popular" header as well - would be good to avoid including `APInt` here as well: should be doable by outlining new `SPIRVConstantBase` constructor into corresponding `.cpp` file

Reply:
Removed. If you don't mind I went by taking an APInt reference.

Reply:
No objections, thanks!

Reply:
Looks like I made it wrong. Interesting that it compiled successfully locally.